src/core/ledger/grainAllocation.js

```diff
@@ -66,17 +66,20 @@ export type AllocationPolicy =
 export type BalancedPolicy = {|
   +policyType: Balanced,
   +budget: G.Grain,
+  +mapping?: Function,
```

Comment:
Should add type information for the inputs / outputs of this function for typesafety